doc/symfony-cache-configuration.rst

```diff
@@ -230,6 +267,96 @@ options through the constructor:
         RewriteEngine On
         RewriteRule .* - [E=HTTP_AUTHORIZATION:%{HTTP:Authorization}]
 
+
+.. _taggablestore:
+
+The TaggableStore
```

Comment:
i think this location makes the taggable store a sub section of user context, which is not correct. i would make it a title on the same level as those, using ~. and lets place it right after purge tags.